kitchen-tests/cookbooks/end_to_end/recipes/windows.rb

```diff
@@ -38,6 +38,11 @@
   action :enable
 end
 
+windows_user_privilege "BUILTIN\\Administrators" do
```

Comment:
@chef-davin Am I missing something here. Isn't the name property the principal not the users?